packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -421,7 +419,7 @@ class _TreemapState extends State<Treemap> {
   /// ----------------------------
   /// ```
   Widget buildTreemap(BuildContext context) {
-    if (widget.rootNode.children.isNotEmpty) {
+    if (widget.rootNode!.children.isNotEmpty) {
```

Comment:
add an assert at the beginning of this method to verify widget.rootNode != null

Reply:
done